packages/quantic/force-app/main/default/lwc/quanticFacetStyles/quanticFacetStyles.css

```
@@ -12,7 +12,7 @@ fieldset {
 .facet__show-less {
   font-size: 12px;
   background-color: white;
-  color: var(--lwc-colorTextIconDefault, #747474);
+  color: var(--lwc-colorTextPlaceholder, #747474);
```

Do you mind if we have 3 levels of fallback instead?

```suggestion
  color: var(--lwc-colorTextPlaceholder, var(--lwc-colorTextIconDefault, #747474));
```